packages/ui/src/components/ClinvarStars.tsx

```diff
@@ -3,13 +3,18 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faStar as faStarSolid } from "@fortawesome/free-solid-svg-icons";
 import { faStar } from "@fortawesome/free-regular-svg-icons";
 
+type ClinvarStarsProps = {
+  num: number;
+  length?: number;
+};
+
 const useStyles = makeStyles((theme) => ({
   star: {
     color: theme.palette.primary.main,
   },
 }));
 
-function ClinvarStars({ num, length = 4 }) {
+export default function ClinvarStars({ num, length = 4 }: ClinvarStarsProps) {
```

Comment:
We don't use `export` in the Component declaration. Please roll back to `export default` after declaration.